<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DIGAMBAR-PC" ignored="0" name="UltimatixLogin" tests="4" failures="1" timestamp="24 Jul 2018 18:03:14 GMT" time="110.425" errors="0">
  <testcase name="verifyLoginPageTitle" time="0.043" classname="com.qa.tests.LoginPageTest"/>
  <testcase name="verifyLogo" time="0.103" classname="com.qa.tests.LoginPageTest"/>
  <testcase name="verifyFooter" time="0.077" classname="com.qa.tests.LoginPageTest"/>
  <testcase name="verifyLogin" time="8.218" classname="com.qa.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [Home : Ultimatix - Digitally Connected !] but found [UltimatixHome]">
      <![CDATA[java.lang.AssertionError: expected [Home : Ultimatix - Digitally Connected !] but found [UltimatixHome]
at com.qa.tests.LoginPageTest.verifyLogin(LoginPageTest.java:46)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
</testsuite> <!-- UltimatixLogin -->
